README.md

```diff
@@ -43,54 +43,76 @@ the [ToolforgeBundle](https://github.com/wikimedia/ToolforgeBundle).
+installed, or if you wish to keep an isolated installation of PHP 7.4 for CopyPatrol.
+The base Docker image used is the `toolforge-php74-sssd-base` image from the Toolforge
+Docker image registry, to ensure an environment as close to Toolforge as possible.
 
-_TODO: Update any of these instructions as necessary following the Symfony migration._
-
-1. Build the Docker image for CopyPatrol.
-   ```bash
-   docker-compose -f docker-compose.dev.yml build
-   ```
-2. Install Composer packages using the PHP version used by the Docker image. Be sure that your current/present working directory is this repository.
-   ```bash
-   # The `wikimedia/copypatrol-development` image is generated in the first step.
-   docker run --rm -it -v $(pwd):/app wikimedia/copypatrol-development composer install
-   ```
-   On Windows, use the following command instead:
-   ```bash
-   docker run --rm -it -v %CD%:/app wikimedia/copypatrol-development composer install
-   ```
-3. Edit the `.env.local` file that was created by composer.
-    1. Use the credentials in your `replica.my.cnf` file in the home directory of your
+1. Copy [.env](.env) to [.env.local](.env.local) and fill in the appropriate details.
+    1. Set `REPLICAS_HOST_*` and `TOOLSDB_HOST` to `127.0.0.1`
+    2. Use the credentials in your `replica.my.cnf` file in the home directory of your
        Toolforge account for `REPLICAS_USERNAME`, `REPLICAS_PASSWORD`, as well as
        `TOOLSDB_USERNAME` and `TOOLSDB_PASSWORD`.
-    2. If you need to test (un)reviewing CopyPatrol cases, `TOOLSDB_USERNAME` and `TOOLSDB_PASSWORD`
+    3. If you need to test (un)reviewing CopyPatrol cases, `TOOLSDB_USERNAME` and `TOOLSDB_PASSWORD`
        need to be set to a user with an installation of the CopyPatrol database (`COPYPATROL_DB_NAME`).
-    3. If you need to test OAuth, obtain tokens by registering a new consumer on Meta at
+    4. If you need to test OAuth, obtain tokens by registering a new consumer on Meta at
        [Special:OAuthConsumerRegistration](https://meta.wikimedia.org/wiki/Special:OAuthConsumerRegistration).
        Alternatively, you can set `LOGGED_IN_USER` to any value to simulate being that user
        after clicking on 'Login'.
-    4. `ITHENTICATE_USERNAME` and `ITHENTICATE_PASSWORD` are not necessary unless you need
+    5. `ITHENTICATE_USERNAME` and `ITHENTICATE_PASSWORD` are not necessary unless you need
        to test the "iThenticate report" functionality.
-4. Open up an SSH tunnel to access the databases on Toolforge. This assumes you have left
-   the `REPLICAS_HOST_*` and `REPLICAS_PORT_*` variables at their defaults.
+2. Build the development image once and install Composer dependencies with the following
    ```bash
-   symfony console toolforge:ssh --bind-address=0.0.0.0 --toolsdb
+   # (optional) Prevent double-downloading when the build occurs.
+   docker image pull docker-registry.tools.wmflabs.org/toolforge-php74-sssd-base:latest
+   docker-compose build
```

Comment:
is the dash intentional? I've seen both, but on my machine I had to use `docker compose`

Reply:
It seems `docker compose` is the new, suggested command. `docker-compose` refers to an old version of Docker Compose.<sup>[[1](https://stackoverflow.com/a/66516826/6011166)]</sup> This doesn't have much effect on Windows, but I guess that's not the case everywhere. I'll have this fixed in a later patch.